src/ucs/sys/string.c

```diff
@@ -124,3 +124,23 @@ void ucs_memunits_to_str(size_t value, char *buf, size_t max)
     }
 }
 
+char* ucs_sockaddr_str(struct sockaddr *sock_addr, char *ip_str)
```

Comment:
need to pass max size to this function and pass that max_size to inet_ntop - instead of passing constants